src/app/(proper_react)/(redesign)/(public)/how-it-works/components/Footer.tsx

```diff
@@ -0,0 +1,29 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { Props } from "../HowItWorksView";
+import { Button } from "../../../../../components/client/Button";
+import styles from "../HowItWorksView.module.scss";
+
+export const Footer = (props: Props) => {
+  const { l10n } = props;
+
+  return (
+    <footer className={styles.footer}>
```

Comment:
We already have a `<footer>` element on the page that is being added by the `Footer` component in Shell [here](https://github.com/mozilla/blurts-server/blob/main/src/app/(proper_react)/(redesign)/Shell.tsx#L89). It is valid by W3 standards to have multiple adjacent footer elements, but I don’t think this section is a footer in the intended sense. From [WHATWG – The footer element](https://html.spec.whatwg.org/multipage/sections.html#the-footer-element):

> The [footer](https://html.spec.whatwg.org/multipage/sections.html#the-footer-element) element [represents](https://html.spec.whatwg.org/multipage/dom.html#represents) a footer for its nearest ancestor [sectioning content](https://html.spec.whatwg.org/multipage/dom.html#sectioning-content-2) element, or for [the body element](https://html.spec.whatwg.org/multipage/dom.html#the-body-element-2) if there is no such ancestor. A footer typically contains information about its section such as who wrote it, links to related documents, copyright data, and the like.

Reply:
Updated here:
https://github.com/mozilla/blurts-server/pull/4632/commits/3009c8e1c2974b945d23acfa7568449cedb34f6a

Reply:
Thanks for updating! My original comment was more towards the use of the `<footer>` element even though semantically it might now act as it is originally intended. Should this rather be a <section>?